R/read_data.R

```diff
@@ -24,16 +24,12 @@ read_csv_regions <- function(csv_file) {
 }
 
 read_csv <- function(file, ...) {
-  header <- readLines(file, 1)
-  if (!grepl(",", header) && grepl(";", header)) {
```

Comment:
does data.table cope with both delimiters ok? do we have tests for this? I presume so...

Reply:
It does, we have a test https://github.com/mrc-ide/hintr/blob/master/tests/testthat/test-read-data.R#L71

Reply:
fab thanks